packages/vite-plugin-cloudflare/package.json

```diff
@@ -25,7 +25,8 @@
 		".": {
 			"types": "./dist/index.d.ts",
 			"import": "./dist/index.js"
-		}
+		},
+		"./package.json": "./package.json"
```

Comment:
Does this need to be a public export?

Reply:
Ah, probably not now as I'm importing the file directly. Previously we were importing via a subpath. I'll remove it.